src/Umbraco.Web/PublishedCache/NuCache/NuCacheSerializerComponent.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Services;
+
+namespace Umbraco.Web.PublishedCache.NuCache
+{
+    /// <summary>
+    /// Rebuilds the database cache if required when the serializer changes
+    /// </summary>
+    public class NuCacheSerializerComponent : IComponent
+    {
+        private const string Nucache_Serializer_Key = "Umbraco.Web.PublishedCache.NuCache.Serializer";
+        private const string JSON_SERIALIZER_VALUE = "JSON";
+        private readonly IPublishedSnapshotService _service;
+        private readonly IKeyValueService _keyValueService;
+        private readonly IProfilingLogger _profilingLogger;
+
+        public NuCacheSerializerComponent(IPublishedSnapshotService service, IKeyValueService keyValueService,IProfilingLogger profilingLogger)
+        {
+            // service: nothing - this just ensures that the service is created at boot time
```

Comment:
remove comment. Would be a good idea to have an init method.

Reply:
I'll check it out after merging